### ISSUE TITLE
Inverted withVerticalLabels and withHorizontalLabels

### ISSUE THREAD

**Issue by lucasreppewelander** [indiespirit/react-native-chart-kit#234]
Hi, there seems to have been a mix between the X and the Y axis.
[X axis is the horizontal line and Y the vertical](https://www.google.com/search?q=x+axis+y+axis&oq=x+axis+&aqs=chrome.4.69i57j0l7.3255j0j7&sourceid=chrome&ie=UTF-8)

but the code in [abstract-chart.js](https://github.com/indiespirit/react-native-chart-kit/blob/master/src/abstract-chart.js#L149) and specific the `renderVerticalLabels` and `renderHorizontalLabel` mixes this up.
I am not sure if this affects something else but when I do a `withHorizontalLabels={false}` my vertical labels (Y axis labels) gets hidden and vice versa.

not a big issue but I don't think you would like this to stay this as there is some wrong the naming then.

**Comment by Hermanya** [comment]
I agree the naming is confusing. Originally it meant that the labels add some meaning to the grid behind the chart. I suppose it'd be better to name the withXAxisLabels... At this point it seems like a very breaking change for not much benefit.

**Comment by lucasreppewelander** [comment]
You don't understand.

`withHorizontalLabels` actually turns off vertical labels.
`withVerticalLabels` actually turns off horizontal labels.

**Comment by Hermanya** [comment]
Vertical could mean Y axis, but it also could mean labels for vertical lines behind the chart (X axis)
